LottieUWP/Windows/Foundation/Rect.cs

```diff
@@ -0,0 +1,279 @@
+﻿using System;
+
+namespace LottieUWP
+{
+    internal struct Point
+    {
+        public Point(double x, double y)
+        {
+            _x = x;
+            _y = y;
+        }
+
+        public double X
+        {
+            get
+            {
+                return _x;
+            }
+            set
+            {
+                _x = value;
+            }
+        }
+        public double Y
+        {
+            get
+            {
+                return _y;
+            }
+            set
+            {
+                _y = value;
+            }
+        }
+        internal double _x;
+        internal double _y;
+    }
+
+    internal struct Rect
+    {
+        static private Rect CreateEmptyRect()
+        {
+            Rect rect = new Rect();
+            // We can't set these via the property setters because negatives widths
+            // are rejected in those APIs.
+            rect._x = 0.0;
+            rect._y = 0.0;
+            rect._width = 0.0;
+            rect._height = 0.0;
+            return rect;
+        }
+
+        static internal Rect InitRectforBoundBox()

```

Comment:
This Rect struct is already implemented by the Windows SDK. All you have to do is use the native Windows.Foundation.Rect struct. You could implement this two methods by using an extension method. Just a static method on the RectExt class (https://github.com/azchohfi/LottieUWP/blob/master/LottieUWP/RectExt.cs).
Also, you don't need to create a new PR for every new commit. Just push the new commits to the same branch and the PR will reflect those new changes.